docs/features/runs/README.md

```diff
@@ -19,6 +19,11 @@ cd hello
 grid run hello.py --number "[1, 2]" --food_item "['pizza', 'hotdog']"
 ```
 
+The Grid CLI also enables seperation of the grid command line arguments from script arguments with the '--' flag. As an example:
```

Comment:
@oojo12 not sure what you're saying here? it's more that grid arguments come before the script and any script arguments come after the script

like `grid run --grid_param argument --grid_param argument train.py --script_param argument --script_param argument`

Reply:
@essiequoi How about:

"The Grid CLI accepts "--" as a separator between grid arguments and script arguments. This is useful under situations where the Grid CLI arguments have the same name as your script arguments, preventing name clashing"

Reply:
yes